dingo/gw/training/train_builders.py

```diff
@@ -271,9 +271,7 @@ def build_svd_for_embedding_network(
         wfd,
         batch_size=batch_size,
         num_workers=num_workers,
-        worker_init_fn=lambda _: np.random.seed(
-            int(torch.initial_seed()) % (2 ** 32 - 1)
-        ),
+        worker_init_fn=fix_random_seeds,
```

Comment:
I think you need to import this, no?

Reply:
It's imported via `from dingo.core.utils import *`. This is maybe not good style, but not part of this PR.

Reply:
Ah, I missed that1